renderer.js

```diff
@@ -69,6 +85,26 @@ class Song {
         return (time - (time %= 60)) / 60 + (9 < time ? ':' : ':0') + time;
     }
 
+    download() {
+        const url = `http://pleer.net/site_api/files/get_url?action=download&id=${this.link}`;
+        try {
+            fetch(url)
+                .then(blob => blob.json())
+                .then(response => {
+                    if (response.success === true && response.residue_type === "tracks") {
+
+                        if (!fs.existsSync(`${userHome}/${MUSIC_DIR}`)) {
```

Comment:
Do you really need a package to get the user home ? I think you cane use something like [this](https://nodejs.org/api/os.html#os_os_homedir) instead of an npm package